quarkus-extension/simple-rest-example/pom.xml

```
@@ -0,0 +1,118 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.acme</groupId>
+  <artifactId>code-with-quarkus</artifactId>
```

Maybe we can rename the coordinates to be in line with the other Camunda examples:

```suggestion
  <groupId>org.camunda.bpm.quarkus.example</groupId>
  <artifactId>camunda-bpm-quarkus-example-simple-rest</artifactId>
```

---

Makes sense 👍